### ISSUE TITLE
[2.5] changes on static, full-width homepage....

### ISSUE THREAD

**Issue by GlennMartin1** [woocommerce/storefront#1127]
Wordpress 5.2.0
Storefront 2.5
Homestore 2.0.31
Powerpack 1.4.15

On my homepage, I have it set as "Full Width"
Also, it is set as a "static" page in the Customizer.

Since upgrading to Storefront 2.5, there's two (related?) issues:
- the homepage title disappeared
- the pics are not wrapping properly as before

Here a **Before** and **After** image:

### BEFORE (Storefront 2.4.6):
![before](https://user-images.githubusercontent.com/30901407/57465077-e0bb5d80-724b-11e9-94ba-4bf0ee9108db.jpg)

### AFTER (Storefront 2.5):
![after](https://user-images.githubusercontent.com/30901407/57465092-e9ac2f00-724b-11e9-891a-0d104daab3a5.jpg)

**Comment by GlennMartin1** [comment]
On 7 sites I've upgraded (to Storefront 2.5) so far, 4 have this issue, 3 do not.

They all have the same site environment (listed in above post). Not clear what the difference is.

All have the homepage content in the new editor.
All running Gutenberg plugin (but same issue with Gutenberg plugin deactivated).

------

Regarding the image wrapping, perhaps this image could be a clue for you:
![after 2](https://user-images.githubusercontent.com/30901407/57467614-d64f9280-7250-11e9-8e84-0ec172ec2470.jpg)

**Comment by tiagonoronha** [comment]
@GlennMartin1 in 2.5 we repurposed the "full width" template as a way to create landing pages without a page title. Sorry if this is causing you trouble.

You can manually add a heading to the post content, it will show basically the same as the title.

If you don't have any widgets in the sidebar, you can also remove the "Full Width" template from that page and the layout should still stay full width, but with the title.

**Comment by GlennMartin1** [comment]
Thanks for replying.

I have no sidebar widgets.

You said:
> If you don't have any widgets in the sidebar, you can also remove the "Full Width" template from that page and the layout should still stay full width, but with the title.

The page template options listed are:
- Default Template
- **Full Width** (currently selected)
- Homepage

Neither **Default Template** nor **Homepage** give the full-width template with title. What am I missing?

Adding an H1 header would require restyling that with CSS to return to the former title.

**Comment by GlennMartin1** [comment]
Leaving aside the **Title** issue on the full-width template, the remaining issue is the faulty image wrapping.

**Comment by GlennMartin1** [comment]
Thanks for your patience with this issue....

Perhaps it's just me - but the "**Homepage**" template produces an empty page - no body content, despite selecting "static content" in the Customizer.

The "**Full-width**" template has no title (shouldn't each page have an H1?). Plus this **template has text-wrap issues**, and an **unwanted white-space** before content begins (area where the title used to be).

Great theme otherwise - not wanting to complain - but the homepage is just not working.

**Comment by tiagonoronha** [comment]
@GlennMartin1 the image wrapping is a different issue that is not related to the changes we made. I'll address it in a different issue.

As for the changes to the title, the change made was to **not** show the title when a page with the "Full Width" template is assigned as the frontpage.

I just tested creating a page with the default template, then I assigned it to be the front page and the title is showing:

<img width="280" alt="Screenshot 2019-05-13 at 12 48 55" src="https://user-images.githubusercontent.com/1177726/57612430-7f61fa00-757d-11e9-931d-7d5e88a693ab.png">
